.github/workflows/rubyonrails.yml

```diff
@@ -26,13 +26,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 100
+          submodules: recursive
       - name: Install Ruby and gems
         uses: ruby/setup-ruby@v1 # v1.115.3
         with:
           bundler-cache: true
       - uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '16'
```

Comment:
@Arnabdaz have you tested the compatibility with this node version?

Reply:
Yes